package.json

```diff
@@ -30,7 +30,7 @@
   "scripts": {
     "test": "jest --passWithNoTests",
     "cypress:open": "cypress open --project ./spec",
-    "cypress:run": "cypress run --project ./spec",
+    "cypress:run": "cypress run --project ./spec --config video=false,screenshotOnRunFailure=false",
```

Comment:
Turns out this doesn't do anything, because the github action doesn't run this. I'll fix it soon, we need to change it in the config file